source/instructors/setupcourses/computedvm.rst

```diff
@@ -0,0 +1,82 @@
+.. meta::
+   :description: Instructions for using Computed VMs.
+
+
+.. _computedvm:
+
+Computed VMs
+============
+
+Codio supports Windows OS in **Computed VM** boxes . To enable this for your organisation, send an email to help@codio.com with approximate details of the number of students who will require access to computed VMs.
+
+.. Note:: Codio are not responsible for licensing issues in regard to any software you install to use.
+
+The computed VM's can support 1, 4, 8 or 16Gb memory.   VM's are enabled for the organisation and then set up at the **Course** level. All assignments in the course would then be able to utilise the computed VM boxes, so we would recommend setting up a new course for those assignments that require them.  All assignments in the course would use the same **VM Stack**.
```

Comment:
The computed VM's can support 1, 4, 8 or 16Gb memory. The VM's are enabled for the organization and then set up at the **Course** level. All assignments in the course can utilize the computed VM boxes, so we would recommend setting up a new course for those assignments that require them. All assignments in the course would use the same **VM Stack**.